### ISSUE TITLE
Evaluate on other dataset?

### ISSUE THREAD

**Issue by Yuliang-Zou** [princeton-vl/relative_depth#2]
How can we evaluate the pre-trained models on other dataset? For example, the Make3D dataset. Can we use the mean and variance of the training set of NYU-D to normalize the predicted output?

**Comment by wfchen-umich** [comment]
I think it may be problematic to use the mean and variance of the NYU dataset on other datasets, considering that the data distribution is different. I suggest you to normalize using the mean and variance of the dataset you want to experiment on.
